src/main/features/response/views/confirmation.njk

```diff
@@ -8,51 +8,74 @@
     <div class="column-two-thirds">
       <div class="govuk-box-highlight">
         <h1 class="bold-large">{{ t("Defence submitted") }}</h1>
-
-        <p>
+        <p class="response-date">
           <strong class="bold">{{ submittedOn | date }}</strong>
         </p>
         <p class="issue-receipt-download ">
-          <a href="{{ paths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t("Download defence as pdf") }}</a>
+          <a href="{{ paths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t("Download your response as PDF") }}</a>
         </p>
       </div>
 
-      <p>
-        {{ t("Your defence will be shared with %s and the court.",
-        {
-          postProcess: 'sprintf',
-          sprintf: [claim.claimData.claimant.name]
-        }
-        ) }}
-      </p>
-      <p>{{ t('We’ve also sent a copy to:') }} <strong>{{ defendantEmail }}</strong>.</p>
+      <div>
+        <h2 class="heading-medium">{{ t("Your response") }}</h2>
+        <p>
+          {{ t(
+            'We’ve emailed %s your defence, explaining why you disagree with the claim.',
+            {
+              postProcess: 'sprintf',
+              sprintf: [claimantName]
+            }
+          ) }}
+        </p>
+        <p>
+          {{ t(
+            'View the <a href="%s">claim against you and your response</a>.',
+            {
+              postProcess: 'sprintf',
+              sprintf: ['/dashboard/' + claim.externalId + '/defendant']
+            }
+          ) | safe }}
+        </p>
+      </div>
 
       <div>
         <h2 class="heading-medium">{{ t("What happens next") }}</h2>
         <p>
-          {{ t("The case will be reviewed by a judge and might go to a hearing. We’ll contact you if a hearing date is scheduled and explain what you need to do to prepare.") }}
+          {{ t(
+            '%s has until %s to respond to your defence.',
+            {
+              postProcess: 'sprintf',
+              sprintf: [claimantName, '!!! TODO Calculate this date !!!']
```

Comment:
Calculate this date?

Reply:
I'm waiting for Art to respond on this. We don't currently have a config for number of days claimant has to respond to a defence.